README.md

```diff
@@ -83,6 +84,90 @@ A transformation is also added to the image - cropping and using the sepia effec
 This will output the following url:
 `https://res.cloudinary.com/demo/image/upload/c_fill,h_150,w_100/sample.jpg`
 
+### Uploading Assets
+
+The entry point for upload operations is the `cloudinary.uploader().upload()` call. All upload operations are dispatched to a background queue.
+
+The following examples uploads a `File` using the default settings, a request upload callback, and an upload preset (more about upload presets below):
+
+```kotlin
+cloudinary.uploader().upload(imageFile) { 
+    params {
+      uploadPreset = "sample_preset"
+    }
+    options {
+      unsigned = true
+    }
+}
+```
+
+The uploaded image is assigned a randomly generated public ID, which is returned as part of the response object.
+The image is immediately available for download through a CDN:
+
+    cloudinary.image().generate("generatedPublicId")
+      
+    http://res.cloudinary.com/demo/image/upload/generatedPublicId.jpg
```

Comment:
maybe replace `demo` with `<your cloud name>` or something like that

Reply:
will do